packages/flutter/lib/src/material/app_bar_theme.dart

```diff
@@ -166,6 +169,9 @@ class AppBarTheme with Diagnosticable {
   /// Overrides the default value of [AppBar.titleTextStyle]
   /// property in all descendant [AppBar] widgets.
   ///
+  /// Please set [backwardsCompatibility] true if you want to override
```

Comment:
I think you want to say that if this property is specified, then backwardsCompatibility should be true. Although one could override the `get titleTextStyle` method, that doesn't seem like the case that's worth documenting.

Reply:
Thanks @HansMuller